config/manifests/bases/awx-resource-operator.clusterserviceversion.yaml

```diff
@@ -34,10 +34,12 @@ spec:
         path: tower_auth_secret
```

Comment:
@neevnuv If you are up to it, another change that is needed is to add the "hidden" x-descriptors to the now deprecated `tower_auth_secret` parameters.

```
      - displayName: AWX Authentication Secret (DEPRECATED)
        path: tower_auth_secret
        x-descriptors:
        - urn:alm:descriptor:com.tectonic.ui:hidden
```

Reply:
Sure, I'll Open for it a new PR